src/EnergyPlus/SingleDuct.hh

```diff
@@ -212,10 +228,20 @@ namespace SingleDuct {
         int ZoneTurndownMinAirFracSchPtr;    // pointer to the schedule for turndown minimum airflow fraction
         Real64 ZoneTurndownMinAirFrac;       // turndown minimum airflow fraction value, multiplier of zone design minimum air flow 
         bool ZoneTurndownMinAirFracSchExist; // if true, if zone turndown min air frac schedule exist
+        bool MyEnvrnFlag;
+        bool MySizeFlag;
+        bool GetGasElecHeatCoilCap;          // Gets autosized value of coil capacity
+        bool PlantLoopScanFlag;              // plant loop scan flag, false if scanned
+        Real64 MassFlow1; // previous value of the terminal unit mass flow rate
+        Real64 MassFlow2; // previous value of the previous value of the mass flow rate
+        Real64 MassFlow3;
+        Real64 MassFlowDiff;
+        SingleDuctAirTerminalFlowConditions sd_airterminalInlet;
+        SingleDuctAirTerminalFlowConditions sd_airterminalOutlet;
```

Comment:
created a nested struct to avoid passing air terminal index as argument to member functions